src/Chord.java

```diff
@@ -0,0 +1,115 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+/**
+ * Allows a user to transverse through a chord.
+ *
+ * @author
+ */
+public class Chord {
+
+    /**
+     * Removes the last note in a chord.
+     *
+     * @param q
+     *
+     * @return removed
+     */
+    public static char removeLast(Queue<Character> q) {
+        q.flip();
+        char removed = q.dequeue();
+        q.flip();
+
+        return removed;
+    }
+
+    /**
+     * Gets the root node of the given chord.
+     *
+     * @param q
+     *
+     * @return root
+     */
+    public static char getRoot(Queue<Character> q) {
+        char root = q.front();
```

Comment:
Interesting idea! I would make sure you mark this in the convention later. Also, this probably doesn't matter much, but isn't it possible for the root to not be the lowest note (i.e., chord inversions)?